search-resources-api/src/test/java/no/unit/nva/search/SearchResourcesApiHandlerTest.java

```diff
@@ -67,23 +71,21 @@ void shouldReturnSearchResultsWhenQueryIsSingleTerm() throws IOException {
         SearchResourcesResponse expected = getSearchResourcesResponseFromFile(ROUNDTRIP_RESPONSE_JSON);
 
         assertNotNull(gatewayResponse.getHeaders());
-        assertEquals(gatewayResponse.getStatusCode(), HttpStatus.SC_OK);
+        assertEquals(HttpStatus.SC_OK, gatewayResponse.getStatusCode());
```

Comment:
I prefer to use HttpUrlConnection codes instead of HttpStatus because then we depend on an apache library for no reason